science/libsbml/Portfile

```diff
@@ -1,12 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cmake 1.0
+
+cmake.out_of_source yes
```

Comment:
Please test and use the cmake 1.1 portgroup instead. See https://trac.macports.org/ticket/47197#comment:63.